Editor/Bootstrap/BootstrapSettings.cs

```diff
@@ -104,43 +104,48 @@ public async Task SetUp(CancellationToken cancellationToken = default)
             await RefreshExistingBuckets(cancellationToken);
         }
 
-        public void SelectBucket(string name)
+        public bool SelectBucket(string name)
```

Comment:
Is the return type important? I don't see any invokers using the return type. If this isn't being used, please revert the change.